shumai/io/network.ts

```diff
@@ -273,7 +280,20 @@ export function serve(request_dict: Record<string, any>, options: ServeOpts) {
     } else {
       ret = await fn()
     }
-    if (ret && ret.constructor === sm.Tensor) {
+
+    if (ret && ret instanceof sm.Tensor) {
+      if (ret.stats !== null) {
+        const segments = req.url.split('/')
+        const last_seg = segments[segments.length - 1]
+        const route = last_seg in request_dict ? last_seg : 'default'
+        if (!(route in statistics)) {
+          statistics[route] = {
+            hits: 0,
+            seconds: 0
+          }
+        }
+        statistics[route].op_stats = ret.stats
```

Comment:
instead of populating `statistics[route]`, maybe populate `op_stats[route]` (created on line 253)